README.md

```diff
@@ -18,24 +23,35 @@ pip3 install -r sphinx/requirements.txt
 python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 ```
 
-## Quickstart Build
+## Build and install
 
-To download the hipSOLVER source code, clone this repository with the command:
+To download the hipSOLVER source code, clone this repository using:
 
-    git clone https://github.com/ROCmSoftwarePlatform/hipSOLVER.git
+```bash
+git clone https://github.com/ROCmSoftwarePlatform/hipSOLVER.git
+```
 
-hipSOLVER requires either cuSOLVER or rocSOLVER + SuiteSparse to be installed on the system. Once these are installed, the following commands will build hipSOLVER and install to `/opt/rocm`:
+To install hipSOLVER, you must have cuSOLVER or rocSOLVER and rocBLAS installed on your system.
```

Comment:
SuiteSparse is also a required dependency as of 6.1. The full dependency tree for hipSOLVER is:
- rocSOLVER
  - rocBLAS
    - Tensile
  - rocSPARSE
    - rocPRIM
- SuiteSparse

Reply:
made an update to this bit--mind reviewing it? I used the same formatting, but can restructure it to be more verbose if that would make more sense...